homeassistant/components/honeywell/__init__.py

```diff
@@ -85,7 +89,7 @@ async def async_setup_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> b
         _LOGGER.debug("No devices found")
         return False
     data = HoneywellData(config_entry.entry_id, client, devices)
-    hass.data.setdefault(DOMAIN, {})[config_entry.entry_id] = data
+    config_entry.runtime_data = data
```

Comment:
might as well combine these 2

Reply:
But that's not the example :)  I was thinking I was missing a python concept there...